actiontext/lib/templates/installer.rb

```diff
@@ -1,3 +1,13 @@
+require "pathname"
```

Comment:
Should this installer still exists? Is not the idea to generate the application and already get actiontext on it like the other frameworks like actioncable?

cc @dhh

Reply:
I'm not sure, TBH. Active Storage needs to be installed first, which requires running `rails active_storage:install` (I think?).

Any reason not to merge this in the meantime?

Reply:
I think we can make that work. Not a blocker for this PR, just asking because this may be something we need to change before the beta2.

Reply:
Yes, the idea is still that this requires running a task if you actually want to use it in the app. Since we do migrations and stuff. Just like active storage. But that everything is already there. Nothing to configure or setup or add dependencies for. Just a single task to run.